### PR TITLE
api(ticdc): status api add cluster id

### LINKED ISSUE #6625: status api add cluster id
add cluster into status api response, so we can get the cluster from cdc Open API

### PR DESCRIPTION
**Pull request by sdojjy**
### What problem does this PR solve?

Issue Number: close #6625

### What is changed and how it works?
add ClusterID field
```
type ServerStatus struct {
	Version   string   `json:"version"`
	GitHash   string   `json:"git_hash"`
	ID        string   `json:"id"`
	ClusterID string   `json:"cluster_id"`
	Pid       int      `json:"pid"`
	IsOwner   bool     `json:"is_owner"`
	Liveness  Liveness `json:"liveness"`
}
```

#### Tests 

 - Unit test
 - Integration test

### Release note 

```release-note
`None`.
```

### CODE REVIEW

File: cdc/api/v1/api_test.go [hunk 1]
```diff
@@ -785,9 +789,11 @@ func TestServerStatus(t *testing.T) {
 	require.Nil(t, err)
 	require.Equal(t, "capture-for-test", resp.ID)
 	require.True(t, resp.IsOwner)
+	require.Equal(t, "abcd", resp.ClusterID)
```
Comment (lonng) [inline cdc/api/v1/api_test.go:792]: Do we need to add `liveness` test cases?

Comment (sdojjy) [reply cdc/api/v1/api_test.go:792]: liveness test cases is already added

**Review by 3AceShowHand — APPROVED** [review]

**Review by asddongmen — APPROVED** [review]

**Pull request merged**